src/control/server/server_utils.go

```diff
@@ -735,21 +735,19 @@ func registerFollowerSubscriptions(srv *server) {
 	srv.pubSub.Subscribe(events.RASTypeStateChange, srv.evtForwarder)
 }
 
-func isSelfHealExcludeSet(svc *mgmtSvc) bool {
-	// Retrieve system self-heal property.
-	selfHeal, err := system.GetUserProperty(svc.sysdb, svc.systemProps, daos.SystemPropertySelfHeal.String())
-	if system.IsErrSystemAttrNotFound(err) {
-		return true // Assume default value where all flags are set.
-	}
+func isSelfHealExcludeSet(svc *mgmtSvc) (bool, error) {
+	selfHeal, err := svc.getSysSelfHeal()
 	if err != nil {
-		svc.log.Errorf("retrieving %s system property: %s", daos.SystemPropertySelfHeal,
-			err.Error())
-		return true // Assume default values if system property cannot be accessed.
+		if system.IsErrSystemAttrNotFound(err) {
+			// Assume default value where all flags are set if property isn't present.
+			return true, nil
```

Comment:
For my understanding, is it for being compliant with old system not having set this property ?